vehicle/mkvi/software/sensing/sensing.c

```diff
@@ -0,0 +1,73 @@
+/*
+Header:
+    Code for the Sensing Controller Board located in the nostril motherboard and
```

Comment:
I'm probably very behind on the current sensing architecture, but is the sensing firmware specific to front and rear boards? If so, probably should create different directories for both

Reply:
They will be the same, but we need to figure out some way to differentiate them so they know which CAN messages to send out. Probably EEPROM makes the most sense. @richardli03 I can work on that once your code is good to go.